src/interventions.c

```diff
@@ -352,6 +353,15 @@ void update_intervention_policy( model *model, int time )
 		model->manual_trace_interview_quota = params->manual_trace_n_workers * params->manual_trace_interviews_per_worker_day;
 		model->manual_trace_notification_quota = params->manual_trace_n_workers * params->manual_trace_notifications_per_worker_day;
 	}
+
+	for( int idx = 0; idx < model->params->n_total; idx++ )
```

Comment:
I'm not sure whether we should be looping over everybody here, it is very inefficient, especially since this is for an intervention and will only apply to a very small number of people. Can you make it event driven?